test/helpers/tenant/secrets.go

```diff
@@ -15,26 +15,30 @@ import (
 	k8serrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/yaml"
+	"sigs.k8s.io/e2e-framework/pkg/env"
 	"sigs.k8s.io/e2e-framework/pkg/envconf"
 	"sigs.k8s.io/e2e-framework/pkg/features"
 )
 
+const (
+	otelSecretName = "dynatrace-operator-otel-config"
+)
+
 var (
 	defaultSingleTenant      = path.Join(project.TestDataDir(), "secrets/single-tenant.yaml")
 	defaultMultiTenant       = path.Join(project.TestDataDir(), "secrets/multi-tenant.yaml")
 	defaultEdgeConnectTenant = path.Join(project.TestDataDir(), "secrets/edgeconnect-tenant.yaml")
+	defaultOtelTenant        = path.Join(project.TestDataDir(), "secrets/otel-tenant.yaml")
 )
 
 type Secrets struct {
 	Tenants []Secret `yaml:"tenants"`
 }
 
 type Secret struct {
-	TenantUid                       string `yaml:"tenantUid"`
-	ApiUrl                          string `yaml:"apiUrl"`
-	ApiToken                        string `yaml:"apiToken"`
-	SyntheticLocEntityId            string `yaml:"syntheticLocEntityId"`
-	SyntheticBrowserMonitorEntityId string `yaml:"syntheticBrowserMonitorEntityId"`
+	TenantUid string `yaml:"tenantUid"`
```

Comment:
Am I wrong or isn't it called TenantUUID?

Reply:
I didn't name that, I just removed/moved that part 😬